scripts/bootstrap/buildbootstrapcli.sh

```diff
@@ -300,11 +299,11 @@ sed -i \
     -e 's/"runtimes": {/&\n    "'$__runtime_id'": [\n      "unix", "unix-'$__build_arch'", "any", "base"\n    ],/g' \
 $__frameworkpath/Microsoft.NETCore.App.deps.json
 
-__crossgentimeout=120
+__crossgentimeout=300
```

Comment:
Original timeout was kept short since crossgen typically took a really long time when paths were incorrect.
With correct paths I am seeing more reasonable crossgen times <40s in release.  Extending the timeout for slower platforms and for debug builds.